apps/term-apply/pkg/server/config.go

```diff
@@ -61,13 +63,25 @@ func NewConfig() Config {
 	}
 	log.Printf("TA_RESUME_PREFIX set to '%s'", s3ResumePrefix)
 
+	ssmHostKeyParam, ok := os.LookupEnv("TA_SSM_HOST_KEY_PARAM")
```

Comment:
A nitpick but I think itd be helpful since its consistent with all the options previously: We should log the values these new env vars are set to.

Look at the above log info `log.Printf("TA_RESUME_PREFIX set to '%s'", s3ResumePrefix)` for example.